unit-tests/live/metadata/test-usb-type-found.py

```diff
@@ -0,0 +1,15 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+import pyrealsense2 as rs
+from rspy import test
+
+
+test.start("Testing USB type can be detected")
+
+dev = test.find_first_device_or_exit()
+
+test.check(dev.supports(rs.camera_info.usb_type_descriptor))
```

Comment:
Please add querying and making sure not `Undefined`

Reply:
With the bug present this check was enough to fail it, but I'll add a check making sure the type is not `Undefined`